pages/pet-details/[id].js

```diff
@@ -242,10 +243,14 @@ export default function PetDetails({ petCollection, onDeletePet }) {
           </StyledDeleteButton>
         </PictureAndDeleteContainer>
         <PetName>{pet.name}</PetName>
-        <StyledUpdateButton>
-          <Link href={`/update?id=${pet.id}&hideButtons=true`}>
+        <StyledUpdateButton disabled={!pet.alive}>
```

Comment:
When you pass a prop to a styled component, you should prefix them with "$", otherwise you will get notifications in your console letting you know that you are passing a non-native HTML attribute to the DOM.

```suggestion
        <StyledUpdateButton $disabled={!pet.alive}>
```